docs/conf.py

```diff
@@ -29,6 +29,7 @@
     'sphinx.ext.autodoc',
     'sphinx.ext.doctest',
     'sphinx.ext.napoleon',
+    'sphinx.ext.todo',
```

Comment:
@adamjstewart there are only a couple TODO notes in the code, I don't think it's worth adding an extension.

Reply:
In that case, should I remove the TODOs in the docs, or convert them to comments? I'll go with the latter unless you prefer the former.